pkg/minikube/cluster/cluster_test.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright (C) 2016 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cluster
+
+import (
+	"testing"
+	"github.com/stretchr/testify/assert"
+	"github.com/minishift/minishift/pkg/minikube/constants"
+	"path/filepath"
+	"fmt"
+)
+
+func TestRemoteBoot2DockerURL(t *testing.T) {
```

Comment:
The test looks basically ok. I kind of like the assert as well. However, you are just testing the successful code path. What are the expectations for when the URL cannot be parsed?

And this really tests just _machineConfig.GetISOFileURI()_ which is fine, but where is the test that _createHypervHost_ works? Right now, I could revert your change to the production code, add your test and everything is still passing. What I am after is a test which fails before you apply your change to the production code and then passes when it is applied.